File: server/Practicum.Core/Models/RoleName.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Practicum.Core.Models
{
    public class RoleName
    {
        public int  Id { get; set; }
        public string Name { get; set; }
        public List<EmployeeRole> roleEmployees { get; set; }
    }
}

File: client/src/app/role-name.ts
import { EmployeeRole } from "./employee/models/employee-role.module"

export class RoleName {

    id: number
    roleName: string
    roleEmployees: EmployeeRole[];

    constructor(id: number, roleName: string, roleEmployees: EmployeeRole[]) {
        this.id = id;
        this.roleName = roleName;
        this.roleEmployees = roleEmployees
    }

}
